proto/cosmos/crypto/v1beta1/keys.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+package cosmos.crypto.v1beta1;
```

Comment:
Maybe this is a case where we don't need `beta`? If we use these message names for addresses we definitely don't want beta...

Reply:
Should we just use `v1` then?